ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -7261,6 +7278,24 @@ let session_subject_identifier_logout_all _printer rpc session_id _params =
     )
     subject_identifiers
 
+let session_count printer rpc session_id _params =
+  let count = Client.Session.count_sessions ~rpc ~session_id in
+  printer (Cli_printer.PMsg (Int64.to_string count))
+
+let session_groupby_originator printer rpc session_id _params =
+  let mapping = Client.Session.group_sessions_by_originator ~rpc ~session_id in
```

Comment:
Is it possible to sort the output? Is there a default sort (I assume by key) but in this case we might be interested in the highest numbers first. This is just an idea - if it's easy.

Reply:
No order, I can add this in, per the highest count?